### ISSUE TITLE
ERA5 interpolation using original scripts

### ISSUE THREAD

**Issue by JulioTBacmeister** [islasimpson/ERA5interp#6]
@islasimpson You pointed to some ERA5 reanalyses regridded to the FV 1x1 L32 grid, e.g., 

/glade/scratch/islas/for/suqin/ERA5_2021/regridded/ERA5_0.9x1.25_L32.cam2.i.2021-06-01-00000.nc

These were done with the original script.  What ERA5 data did you use as input?

**Comment by islasimpson** [comment]
I used different ERA5 input from the RDA one.  I used ERA5 input that I downloaded myself.  It's here...

/glade/scratch/islas/for/suqin/ERA5_2021/native

and I did the regridding here...

/glade/u/home/islas/for/suqin/regridera5

**Comment by JulioTBacmeister** [comment]
Thanks Isla,
So do you or @brianpm have any hints on reading grib files in Python - grib has always been one my faves so transparent and easy to use

**Comment by islasimpson** [comment]
I think you can do an open_mfdataset call and add engine='cfgrib'.  I have some examples using the JRA55 data which is all in grib format here.../glade/u/home/islas/sortjra55/

**Comment by JulioTBacmeister** [comment]
Yes I've tried that. I'm told cfgrib is not installed (npl-2022b). So I'm trying to install it in my own clone of npl-2022b

**Comment by JulioTBacmeister** [comment]
FYI this is the error I get
ValueError: unrecognized engine cfgrib must be one of: ['netcdf4', 'h5netcdf', 'scipy', 'pydap', 'store', 'zarr']

**Comment by islasimpson** [comment]
ah ok.  I think I do have cfgrib installed in the environment I was using when I used that script.  /glade/u/home/islas/python/envs/ecpaperenv.yml

**Comment by JulioTBacmeister** [comment]
OK Thanks Isla. Still waiitng for my 'conda install' to finish ... not looking good so far

**Comment by JulioTBacmeister** [comment]
"failed with initial frozen solve. Retrying with flexible solve". ...
I've yet to see anything succeed with the "flexible solve"
